ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -2067,11 +2065,9 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = BetterSocial/BetterSocial.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 301;
-				DEVELOPMENT_TEAM = J22M3M9VYU;
-				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = J22M3M9VYU;
+				DEVELOPMENT_TEAM = "";
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `CODE_SIGN_IDENTITY`, `CODE_SIGN_STYLE`, and `DEVELOPMENT_TEAM` have been changed. If these changes are intentional and you're planning to set these values elsewhere or manually before each build, then it's fine. Otherwise, this could cause issues with code signing.

```diff
- "CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
- CODE_SIGN_STYLE = Manual;
- DEVELOPMENT_TEAM = J22M3M9VYU;
+ CODE_SIGN_IDENTITY = "Apple Development";
+ CODE_SIGN_STYLE = Automatic;
+ DEVELOPMENT_TEAM = "";
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->